src/index.html

```diff
@@ -11,7 +11,194 @@
+            name="name"
+            id="name"
+            pattern="\w+"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="inputs" >
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="12"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+
+        <div class="inputs" >
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="birth"
+            id="birth"
+            required
+          >
+        </div >
+
+        <div class="inputs">
+          <label for="agrrment">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agrrment"
+            id="agrrment"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="inputs" >
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="inputs" >
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="inputs" >
+          Do you love cats?
+          <label for="yes">
+            <input
+              type="radio"
+              name="cats"
+              id="yes"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label for="no">
+            <input
+              type="radio"
+              name="cats"
+              id="no"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="inputs" >
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+
+        <div class="inputs" >
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time"
+            id="time"
+          >
+        </div>
+
+        <div class="inputs" >
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+          name="cars"
+          id="cars"
+          multiple
+          >
+            <option value="#bmw">BMW</option>
+            <option value="#audi">Audi</option>
+            <option value="#lada">Lada</option>
```

Comment:
These hashes in values are redundant
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```